epicure-frontend/src/components/Chef/Chef.scss

```diff
@@ -0,0 +1,161 @@
+@import '../../styles.scss';
+
+.chef-container {
+    margin-left: 19px;
+
+    .title {
+        @include font-style;
+        font-size: 18px;
+        line-height: 35px;
+        letter-spacing: 1.25px;
+        text-align: left;
+
+        @include desktop {
+            font-size: 30px;
+            text-align: center;
+
+        }
+    }
+
+    .image-description-container {
+        display: flex;
+        flex-direction: column;
+
+        @include desktop {
+            flex-direction: row;
+            margin-left: 20px;
+            margin-right: 129px;
+            margin-top: 80px;
+            margin-bottom: 45px;
```

Comment:
Why are your numbers so big and specific for margin? 
Try to use other measures of css